base/applications/mscutils/servman/lang/bg-BG.rc

```diff
@@ -332,6 +331,13 @@ END
 STRINGTABLE
 BEGIN
     IDS_APPNAME "Управление на услугите на РеактОС"
+    IDS_MANAGER_ERR_UNKCMD "Service Error: Unknown command"
+    IDS_MANAGER_ERR_SRVMGROPN "Service Error: Unable to open service manager"
```

Comment:
These messages can be formatted by the system from error codes, no need to add them.

Reply:
Yes (ongoing)